### PR TITLE
Project documented

### PR DESCRIPTION
**Pull request by NilinC**
* README 
* Docker environment updated
* Makefile

### CODE REVIEW

File: app/js/server/server.js [hunk 1]
```diff
@@ -25,7 +18,7 @@ app.get('/api/elems/get', function (req, res) {
 });
 
 app.get('/api/users/get', function (req, res) {
```
Comment (desaintvincent) [inline app/js/server/server.js]: not needed anymore, and `users.json` should be deleted

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
 
-# Install app dependencies
-# A wildcard is used to ensure both package.json AND package-lock.json are copied
-# where available (npm@5+)
-COPY package*.json ./
+RUN apk add --update make
+RUN make up
 
-COPY gulpfile.babel.js ./
+FROM node:alpine AS builder-back
```
Comment (wixyvir) [inline Dockerfile]: This build stage is useless

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+#########
+
+$(NODE_MODULES): package.json package-lock.json
+	$(NPM) install --ignore-scripts && $(NPM) rebuild node-sass
+
+#########
+
+$(DIST_DIR)/dashboard.js: $(DIST_DIR) $(NODE_MODULES) $(ALL_SRC_JS)
+ifeq ($(DASHBOARD_ENV),dev)
+	$(BIN_DIR)/browserify $(SRC_JS)/client/client.jsx --debug --outfile $(DIST_DIR)/dashboard.js --transform [babelify]
```
Comment (wixyvir) [inline Makefile]: to be consistent, maybe define a `$(BROWSERIFY)` variable?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Dashboard
+
+Dashboard provides an interface to monitor job's status for continuous integration.
```
Comment (wixyvir) [inline README.md]: Should'nt we list what are these jobs ? Jenkins...

**Review by wixyvir — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* [Docker](https://docs.docker.com/)
+* [npm](https://www.npmjs.com/get-npm)
+* [NodeJS](https://nodejs.org/en/)
+
+## Getting started
+```
+$ git clone https://github.com/pmsipilot/dashboard.git
+$ cd dashboard/
+
+# This task will create the docker image for the dashboard, install js dependancies and build css
```
Comment (Gwalchmei) [inline README.md]: I prefer smaller code blocks with only useful commands and the comments as plain text. It is easier to read and you can easily copy/paste to get started

File: Dockerfile (hunk 2 above)
Comment (kevinbalicot) [reply Dockerfile]: why ?

Comment (wixyvir) [reply Dockerfile]: Has been fixed

**Review by wixyvir — APPROVED** [review]

**Review by desaintvincent — APPROVED** [review]

**Pull request merged**